prody/dynamics/plotting.py

```diff
@@ -228,9 +228,11 @@ def showProjection(ensemble, modes, *args, **kwargs):
 
     import matplotlib.pyplot as plt
 
+    returnAx = kwargs.pop('returnAx',False)
+
     cmap = kwargs.pop('cmap', None)
 
-    if SETTINGS['auto_show']:
+    if kwargs.pop('new_fig',True):
```

Comment:
We are now abandoning `new_fig` and control the behavior by `auto_show`. If `confProDy(auto_show=True)` then a new figure will also be created.

Reply:
No, it doesn't work like that. I've tested it lots of times. If auto_show is False, you don't get anything until you put plt.show() and when you do you still get multiple figures all the time.

Reply:
Interesting. But I think the problem might be something else. I will look into this.

Did you use `ion` of `ioff`?

Reply:
Neither. I don't know what's the default.

Reply:
Thanks

Reply:
Try replacing `fig, ax = plt.subplots` to `fig = plt.figure`, that might fix it

Reply:
Oh, forget to ask, in your code did you `show` everytime you make a subplot? Because if that's the case then multiple figures will be created. Calling `show` at the very end should fix the problem.

Reply:
Ah ok. That makes sense as long as auto_show is False. We are now doing that.

Reply:
ok we just tested it and that works perfectly